src/converters/fromZigbee.ts

```diff
@@ -2253,6 +2253,21 @@ const converters1 = {
             return {action: `${button}${clickMapping[msg.data[3]]}`};
         },
     } satisfies Fz.Converter,
+    tuya_on_off_action_8: {
+        cluster: 'genOnOff',
+        type: 'raw',
+        convert: (model, msg, publish, options, meta) => {
+            if (hasAlreadyProcessedMessage(msg, model, msg.data[1])) return;
+            const clickMapping: KeyValueNumberString = {0: 'single', 1: 'double', 2: 'hold'};
+            const buttonMapping: KeyValueNumberString = {1: '1', 2: '2', 3: '3', 4: '4', 5: '5', 6: '6', 7: '7', 8: '8'};
```

Comment:
Please merge this with `tuya_on_off_action` above

Reply:
@waterlou I saw you removed the original content, instead a new `else if` statement should be added for this device.